src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -736,6 +736,8 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
 
             should_fuse |= input.is_type<strided_slice>();
 
+            should_fuse |= input.is_type<crop>();
```

Comment:
Could you please clarify why fusions are added here for crop, but actual fusion logic implemented for slice kernel? Is crop going to be replaced with slice at some point of graph optimizations? 
This fusion should probably be carefully checked to ensure we wouldn't prevent any further crop optimizations